modules/management-console-agent/src/test/java/org/apache/ignite/agent/action/controller/AbstractActionControllerTest.java

```diff
@@ -107,6 +112,42 @@ protected void executeAction(Request req, Function<List<JobResponse>, Boolean> a
         executeActionAndStopNode(req, 0, 0, assertFn);
     }
 
+    /**
+     * @param reqId Request ID.
+     */
+    protected List<TaskResponse> taskResults(UUID reqId) {
+        return interceptor.getAllPayloads(buildActionTaskResponseDest(cluster.id()), TaskResponse.class).stream()
+            .filter(r -> r.getId().equals(reqId))
```

Comment:
`reqId.equals(r.getId())`
Expected goes first
In case of nullable there's a risk of NPE